pkg/ddl/ddl_worker.go

```diff
@@ -425,11 +421,10 @@ func (d *ddl) addBatchDDLJobs(tasks []*limitJobTask) error {
 	}
 
 	ctx := kv.WithInternalSourceType(context.Background(), kv.InternalTxnDDL)
-	// lock to reduce conflict
-	d.globalIDLock.Lock()
+
+	ids, err = d.genGlobalIDs(len(tasks))
```

Comment:
seems this optimize only works when we connects to multiple Nodes to do `create-table`

have you any data on how much this can improve performance?

Reply:
![img_v3_02cg_769e1e9e-dd6e-40a5-9ea6-b5ac4596e8dg](https://github.com/pingcap/tidb/assets/35025882/27d0bc22-d167-4bae-9eb0-2d29d9b78d96)
to
![img_v3_02cc_cb333fa6-74f9-41c4-b4c5-ff62c7249f9g](https://github.com/pingcap/tidb/assets/35025882/8c385d83-6ef7-4acf-9922-6b542cc2ef96)